src/index.js

```diff
@@ -217,6 +217,12 @@ module.exports = options => {
       return userFunc;
     }
 
+    if (!config.enabled) {
+      console.debug('IOpipe agent disabled, skipping reporting');
```

Comment:
`console.debug` is not available in node. Also, I think we should probably not output anything here, as disabling the agent is opt-in, so a user should have a path to troubleshoot not seeing invocations in IOpipe pretty easily by inspecting config. We _could_ start using debuglog here though, as that doesn't output anything to stdout unless opted into:
https://nodejs.org/api/util.html#util_util_debuglog_section

One more thing, we should probably move this guard above the token guard.

Reply:
Was bouncing between node.js and python agents to make sure these behaved the same and just copied the equivalent `logger.debug` from the python agent. Updated to use `debuglog`, do we want to use this for any block of code gated by the `debug` flag?